.github/workflows/devcontainer-feature-release.yml

```diff
@@ -0,0 +1,76 @@
+name: Build dev container feature image
+
+env:
+  IMAGE_NAME: ${{ github.repository }}
+  REGISTRY: ghcr.io
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      matrix:
+        baseImage:
+          [
+            "ubuntu:focal",
+            "ubuntu:jammy",
+            "ubuntu:bionic",
+            "debian:11",
+            "debian:10",
+            "mcr.microsoft.com/devcontainers/base:ubuntu",
+            "mcr.microsoft.com/devcontainers/base:debian",
+          ]
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
v3 is the hotness now

Reply:
Probably haven't updated since the original scaffolding and that was out of date.